config/locales/two_factor_authentication/en.yml

```diff
@@ -114,6 +114,12 @@ en:
     phone_fee_disclosure: Message and data rates may apply.
     phone_info_html: We’ll send you a one-time code <strong>each time you sign in</strong>.
     phone_label: Phone number
+    phone_verification:
+      troubleshooting:
+        change_phone_number: Use another phone number
+        code_not_received: I didn’t receive my one-time code
+        header: 'Having trouble? Here’s what you can do:'
```

Comment:
This text already exist for the troubleshooting components as a default of sorts (although I don't see that the defaulting actually happens anywhere 😬 ). Maybe we could point to that instead of duplicating it?

https://github.com/18F/identity-idp/blob/68c41c4e075b3176865907e78c6764a0892e2fb3/config/locales/components/en.yml#L49